workflow/scripts/cnv_json.py

```diff
@@ -140,6 +140,36 @@ def parse_gatk_segments(file):
     return segments
 
 
+@cnv_parser("tsv", header=True)
+def parse_jumble_ratios(file):
+    ratios = []
+    for line in file:
+        ratios.append(
+            dict(
+                chromosome=f"chr{line[0]}",
```

Comment:
The files in the integration test already have the ratios and the segment chromosomes prefixed with `chr`, so this breaks the test for me with a KeyError on e.g. `"chrchrA"`.